packages/helpers/src/tests/handle-visibility-condition-changes.spec.ts

```diff
@@ -6,6 +6,7 @@ import findAsset from '../find-asset'
 
 const payload: MessageVisibilityChanges = {
   messageType: VISIBILITY_CHANGES,
+  assetId: 'C06',
   visibility: [
     {
       assetId: 'C06',
```

Comment:
you should remove the above assetId - since the `visibility` interface has changed

Reply:
Not getting you, as you have asked me to add assetId there.

Reply:
The new line 9 is valid. We don't need the duplicate `assetId` in the new line 12.